### ISSUE TITLE
Toggling WYSIWYG on file will move file to wrong panel

### ISSUE THREAD

**Issue by Blayr** [adamerose/vscode-markdown-wysiwyg#11]
When toggling the editor, it seems to always move the file that is focused to the wrong panel (in my example video, it's the first panel).

https://github.com/adamerose/vscode-markdown-wysiwyg/assets/17113158/0b2841f1-8353-43b9-a1bf-1094a2dae5f6

**Comment by adamerose** [comment]
Yes this is annoying and I'm not sure how to fix it. I couldn't find anything in the VS Code API to directly reopen a file with a different custom editor... `workbench.action.reopenWithEditor` does this and maintains panel positioning, but it shows the user a dialog to pick the editor and there is no similar command that does this without a prompt.

I made an upstream feature request over a year ago about this, go upvote it to increase visibility 😊
https://github.com/microsoft/vscode/issues/169921
